packages/design-system/src/styles/components/_TooltipIcon.scss

```diff
@@ -20,8 +20,7 @@
   fill: $tooltip-icon__color;
   height: $tooltip-icon-size;
   left: 0;
-  // fixes clipping issue in Firefox, overrides vendor style
-  overflow: visible !important;
+  overflow: visible !important; /* stylelint-disable-line declaration-no-important -- fixes clipping issue in Firefox, overrides vendor style */
```

Comment:
This CSS had a very specific comment explaining why the `!important` was needed, but I couldn't find an independent source saying this was still an issue with Firefox (found some indicator it was true with Safari however).

Local testing removing this rule didn't show any obvious breaking issues and I think it _could_ be removed, but want to revisit later with an official ticket to do better browser testing with a refactor.